src/config.cpp

```
@@ -85,14 +85,10 @@ namespace hdt
 					ActorManager::instance()->m_skinNPCFaceParts = reader.readBool();
 				} else if (reader.GetLocalName() == "disableSMPHairWhenWigEquipped") {
 					ActorManager::instance()->m_disableSMPHairWhenWigEquipped = reader.readBool();
-				} else if (reader.GetLocalName() == "clampRotations") {
-					SkyrimPhysicsWorld::get()->m_clampRotations = reader.readBool();
-				} else if (reader.GetLocalName() == "rotationSpeedLimit") {
-					SkyrimPhysicsWorld::get()->m_rotationSpeedLimit = reader.readFloat();
-				} else if (reader.GetLocalName() == "unclampedResets") {
-					SkyrimPhysicsWorld::get()->m_unclampedResets = reader.readBool();
-				} else if (reader.GetLocalName() == "unclampedResetAngle") {
-					SkyrimPhysicsWorld::get()->m_unclampedResetAngle = reader.readFloat();
+				} else if (reader.GetLocalName() == "clampRotations" || reader.GetLocalName() == "rotationSpeedLimit" || reader.GetLocalName() == "unclampedResets" || reader.GetLocalName() == "unclampedResetAngle") {
```

Clang (14.0.6)

[note] +1, nesting level increased to 3

(clang)